pip_audit/_dependency_source/requirement.py

```diff
@@ -240,24 +199,16 @@ def _fix_file(self, filename: Path, fix_version: ResolvedFixVersion) -> None:
             # To know whether this is the case, we'll need to resolve dependencies if we haven't
             # already in order to figure out whether this subdependency belongs to this file or
             # another.
-            if not fixed:
-                req_dep = cast(RequirementDependency, fix_version.dep)
-                if req_dep.dependee_reqs:
-                    logger.warning(
-                        "added fixed subdependency explicitly to requirements file "
-                        f"{filename}: {fix_version.dep.canonical_name}"
-                    )
-                    dependee_reqs_formatted = ",".join(
-                        [
-                            str(req)
-                            for req in sorted(list(req_dep.dependee_reqs), key=lambda x: x.name)
-                        ]
-                    )
-                    print(
-                        f"    # pip-audit: subdependency fixed via {dependee_reqs_formatted}",
```

Comment:
Unfortunately, I don't think we can support this annotation anymore now that we don't have raw access to the resolver.